auth0/src/main/java/com/auth0/android/authentication/AuthenticationException.java

```diff
@@ -85,6 +88,12 @@ public AuthenticationException(Map<String, Object> values) {
         }
     }
 
+    private void warnIfOIDCError() {
+        if ("invalid_request".equals(getCode()) && getDescription().startsWith("OIDC conformant clients cannot use")) {
```

Comment:
Let's mot ever match description values of errors let's print the error and the recommendation if it's invalid_request